src/sentry/grouping/enhancer/__init__.py

```diff
@@ -37,6 +39,8 @@
 DATADOG_KEY = "save_event.stacktrace"
 logger = logging.getLogger(__name__)
 
+RUST_CACHE = RustCache(1_000)
```

Comment:
Whats the reason for 1000? Can you add a comment? Maybe a small sentence with the size of 1 entry in cache would be helpful for future.

Reply:
It's more or less picked out of a hat for now. In my opinion the primary advantage of exposing this to Python at all is so we can tweak the value relatively easily. Eventually we may just make this an implementation detail of the Rust crate and not expose it at all.